src/Microsoft.Net.Http.Headers/DateTimeFormatter.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+
+namespace Microsoft.Net.Http.Headers
+{
+    internal static class DateTimeFormatter
+    {
+        private static readonly DateTimeFormatInfo FormatInfo = CultureInfo.InvariantCulture.DateTimeFormat;
+        private static readonly string[] DayNames = FormatInfo.AbbreviatedDayNames;
+        private static readonly string[] MonthNames = FormatInfo.AbbreviatedMonthNames;
+
+        // The format is "ddd, dd MMM yyyy HH:mm:ss GMT".
+        private const int Rfc1123DateLength = 29;
```

Comment:
nitpicking but you could have done `Rfc1123DateLength = "ddd, dd MMM yyyy HH:mm:ss GMT".Length`